src/pages/workspaces/workspace/Data.js

```diff
@@ -526,6 +530,171 @@ const EntitiesContent = ajaxCaller(class EntitiesContent extends Component {
+  }
+
+  render() {
+    const { workspace, workspace: { accessLevel, workspace: { namespace, bucketName } } } = this.props
+    const { prefix, prefixes, objects, loading, uploading, deletingName, viewingName } = this.state
+    const prefixParts = _.dropRight(1, prefix.split('/'))
+    const canEdit = Utils.canWrite(accessLevel)
+    return h(Fragment, [
+      h(Dropzone, {
+        disabled: !canEdit,
+        disableClick: true,
+        style: { flexGrow: 1, backgroundColor: 'white', border: `1px solid ${colors.gray[3]}`, padding: '1rem' },
+        activeStyle: { backgroundColor: colors.blue[3], cursor: 'copy' },
+        ref: this.uploader,
+        onDropAccepted: files => this.uploadFiles(files)
+      }, [
+        div([
+          _.map(({ label, target }) => {
+            return h(Fragment, { key: target }, [
+              linkButton({ onClick: () => this.load(target) }, [label]),
+              ' / '
+            ])
+          }, [
+            { label: 'Files', target: '' },
+            ..._.map(n => {
+              return { label: prefixParts[n], target: _.map(s => `${s}/`, _.take(n + 1, prefixParts)).join('') }
+            }, _.range(0, prefixParts.length))
+          ])
+        ]),
+        div({ style: { margin: '1rem -1rem 1rem -1rem', borderBottom: `1px solid ${colors.gray[5]}` } }),
+        h(SimpleTable, {
+          columns: [
+            { size: { basis: 24, grow: 0 }, key: 'button' },
+            { header: h(HeaderCell, ['Name']), size: { grow: 1 }, key: 'name' },
+            { header: h(HeaderCell, ['Size']), size: { basis: 200, grow: 0 }, key: 'size' },
+            { header: h(HeaderCell, ['Last modified']), size: { basis: 200, grow: 0 }, key: 'updated' }
+          ],
+          rows: [
+            ..._.map(p => {
+              return {
+                name: h(TextCell, [
+                  linkButton({ onClick: () => this.load(p) }, [p.slice(prefix.length)])
+                ])
+              }
+            }, prefixes),
+            ..._.map(({ name, size, updated }) => {
+              return {
+                button: linkButton({ onClick: () => this.setState({ deletingName: name }) }, [
+                  icon('trash', { size: 16, className: 'hover-only' })
+                ]),
+                name: h(TextCell, [
+                  linkButton({ onClick: () => this.setState({ viewingName: name }) }, [
+                    name.slice(prefix.length)
+                  ])
+                ]),
+                size: filesize(size, { round: 0 }),
+                updated: Utils.makePrettyDate(updated)
+              }
+            }, objects)
+          ]
+        }),
+        deletingName && h(DeleteObjectModal, {
+          workspace, name: deletingName,
+          onDismiss: () => this.setState({ deletingName: undefined }),
+          onSuccess: () => {
+            this.setState({ deletingName: undefined })
+            this.load()
+          }
+        }),
+        viewingName && h(UriViewer, {
```

Comment:
TOL: we should add `.log` to the list of acceptable file extensions.

Reply:
For previewing? It looks like it's already in there.